pkg/bindings/system/info.go

```diff
@@ -6,10 +6,11 @@ import (
 
 	"github.com/containers/podman/v5/libpod/define"
 	"github.com/containers/podman/v5/pkg/bindings"
+	"github.com/containers/podman/v5/pkg/domain/entities/types"
 )
 
 // Info returns information about the libpod environment and its stores
-func Info(ctx context.Context, _ *InfoOptions) (*define.Info, error) {
+func Info(ctx context.Context, _ *InfoOptions) (*types.SystemInfoReport, error) {
```

Comment:
bindings are a stable API we cannot change the types here.

Maybe it would be cleaner to leave the info endpoint and API side as it and move the clientVersion entirely into cmd/podman/system/info.go? i.e. wrap the struct only there. Then we only change the cli level which should be fine